src/NumericalAlgorithms/LinearOperators/Python/PowerMonitors.cpp

```diff
@@ -22,12 +22,34 @@ void bind_power_monitors_impl(py::module& m) {  // NOLINT
           &power_monitors<Dim>),
         py::arg("input_data_vector"), py::arg("mesh"));
 }
+
+template <size_t Dim>
+void bind_relative_truncation_error_impl(py::module& m) {  // NOLINT
```

Comment:
no need for these extra functions, you can just put these `m.def` calls in `bind_power_monitors_impl`.

Reply:
Right. I'll remove them.